src/App.vue

```diff
@@ -4,17 +4,25 @@ import AttributeSection from "./components/AttributeSection.vue";
 import SkillSection from "./components/SkillSection.vue";
 import DisciplineSection from "./components/DisciplineSection.vue";
 import HoverWindow from "./components/HoverWindow.vue";
-import { biography, skillDistributions, clans, attributes, skills, disciplines } from "./data.js";
+import VitalsSideBar from './components/VitalsSideBar.vue'
+import { biography, skillDistributions, clans, attributes, skills, disciplines, vitals, powerStats, resonances } from "./data.js";
 
 export default {
   data() {
     return {
+      /**
+       * load externals
```

Comment:
tu je asi na mieste pouzit jednoriadkovy komentar 
// load externals

Reply:
predpokladam, ze jednoriadkovy komentar pouzit nechces... chapem to tak, ze to tam chces zvyraznene.

```
/**
* docstring - automaticky generuje popis funkcie
**/ 
```
viacriadkovy komentar:
/*
prvy riadok komentara
druhy riadok komentara
*/
```

myslim, ze toto neni docstring, takze by sa mala pouzit dvojriadkova varianta. 

Osobne by som dal jednoriadkovy komentar a ak sa ti to zda malo tak este priazdny riadok

Reply:
ja som to mal len jako odlisenie a preto su tam 3 riadky a nejaky komentar

Reply:
mozem ja za to ze vs automaticky hodi `/** */ `ked napises `/*` a zacina riadky `*` ked das `enter`

Reply:
hej, presne tak, ty za to mozes! :-)

Reply:
napis len 
/* 
a bude to robit, co chces...